doc.html

```diff
@@ -47,104 +47,79 @@ <h2>4 produce_demand_forecaster Usage Scenarios</h2>
 
-It could happen that there is no data available for the requested parameters at the specified boundary and time interval. A common example for that could be that sensors for the requested parameters were never installed within the boundaries, or they were not online during the time interval.
+<h2>5.1 Invalid GEO code</h2>
 
-In those cases, an error message will be returned to the request.
+An error code 404 with an error message will be returned to the request.
 
 <h2>5.2 Invalid time interval</h2>
 
-Sometimes the requested time interval may be invalid according to the available sensor data. For example, if the time interval is 7 minutes and only time series of 1 hour are available.
-
-In all those cases an error would be returned to the request.
-
-To solve the most common requests, the next default time intervals should be covered by the produce_demand_forecaster service implementor: 1 Hour, 1 day, 1 week, 1 month and their multiples.
+An error code 400 with an error message will be returned to the request.
 
 <h2>6 Access and Authentication</h2>
 
-Farmers MUST have an account setup on an produce_demand_forecaster service in order to authenticate and access API endpoints. The service implementor is responsible for the creation of accounts; it is not covered in the service template specifications.
+The produce_demand_forecaster is publically available with rate limits.
+
+For less strict rate limits, producers and retailers and  MUST have an account setup on an produce_demand_forecaster service in order to authenticate and access API endpoints. 
 
-Unless specifically documented in the OpenAPI specifications, all API calls must include credentials in form of Bearer authentication (also called token authentication). Clients can obtain an access token on behalf of their user from the service's authorization server (see ATLAS service pairing).
+To obtain credentials that give access to the endpoints, reach out to the Calabi staff
+at info@calabi.be.
 
-Some of the information held by advisor services may be considered sensitive from a GDPR perspective. The service's authorization server SHOULD request the client's end-user consent at service pairing time in order to deliver an access token.
+Unless specifically documented in the OpenAPI specifications, all API calls must include
+credentials in form of Bearer authentication (also called token authentication). Clients
+can obtain an access token on behalf of their user from the service's authorization
+server (see ATLAS service pairing).
 
-<h2>ANNEX 1 - produce_demand_forecaster Measurement Types</h2>
+<h2>ANNEX 1 - produce_demand_forecaster GEO codes</h2>
 
-The current list of standardized measurement parameters that may be returned by produce_demand_forecaster services is listed below. Newer versions of the produce_demand_forecaster template are likely to extend this list.
+The current list of supported GEO codes. The list is often updated, and the most recent
```

Comment:
GEO codes should follow an established standard so that they remain vendor-independent